bin/i18n/i18n_script_utils.rb

```diff
@@ -383,11 +383,11 @@ def self.sanitize_file_and_write(loc_path, dest_path)
   end
 
   def self.create_progress_bar(**args)
-    ProgressBar.create(**args, format: PROGRESS_BAR_FORMAT)
+    ProgressBar.create(format: PROGRESS_BAR_FORMAT, **args)
```

Comment:
Do we need to update sync-in-base and sync-out-base scripts after this change in argument order os is not necessary?

Reply:
This change just allows us to replace the default `format: PROGRESS_BAR_FORMAT` if needed.
https://github.com/code-dot-org/code-dot-org/blob/65029dd3d441b6a34066f9aa62311b7155a8fd3b/bin/i18n/utils/sync_up_base.rb#L46
https://github.com/code-dot-org/code-dot-org/blob/65029dd3d441b6a34066f9aa62311b7155a8fd3b/bin/i18n/utils/sync_up_base.rb#L79-L85